src/config.ts

```diff
@@ -36,8 +43,12 @@ export const tradingTokens = [
  * if you have deployed your own contract, you can use it instead of the default one
  */
 export const flashloanAddress: string =
-  "0x0e0Ec2C716b5Ba2512Af9d3790D8804Da42DDD58";
+  "0xe43A0003955f8745c77A2A987Afa316D6B9828B3";
```

Comment:
Wouldn't it be wiser to leave @yuichiroaoki smart contract address as a default one?

Reply:
Yes, that would probably be better. I will change it back. Thanks for pointing it out!